### ISSUE TITLE
Don't close event loop passed as argument

### ISSUE THREAD

**Issue by marcinmilewski1** [CircleUp/aresponses#25]
Hi. I have a proposal to don't close the loop when there is no match.
Code snippet:
```
    async def _find_response(self, request):
        host, path, path_qs, method = request.host, request.path, request.path_qs, request.method
        logger.info(f"Looking for match for {host} {path} {method}")
        i = 0
        host_matched = False
        path_matched = False
        for host_pattern, path_pattern, method_pattern, response, match_querystring in self._responses:
            if _text_matches_pattern(host_pattern, host):
                host_matched = True
                if (not match_querystring and _text_matches_pattern(path_pattern, path)) or (
                    match_querystring and _text_matches_pattern(path_pattern, path_qs)
                ):
                    path_matched = True
                    if _text_matches_pattern(method_pattern, method.lower()):
                        del self._responses[i]

                        if callable(response):
                            if asyncio.iscoroutinefunction(response):
                                return await response(request)
                            return response(request)

                        if isinstance(response, str):
                            return self.Response(body=response)

                        return response
            i += 1
        self._exception = Exception(f"No Match found for {host} {path} {method}.  Host Match: {host_matched}  Path Match: {path_matched}")
        self._loop.stop()
        raise self._exception  # noqa

```
This can cause errors like `RuntimeError: Event loop stopped before Future completed.` if this method was called in coroutine.

**Comment by brycedrennan** [comment]
Sweet I’ll check it out. Do you want to make a pull request?

**Comment by marcinmilewski1** [comment]
Yes, I will do.

**Comment by brycedrennan** [comment]
@marcinmilewski1 This is addressed in upcoming 2.0.  Feedback on the PR welcome: https://github.com/CircleUp/aresponses/pull/45
